int32 type
---
float32 error_x  # x偏差
float32 error_y  # y偏差